test/e2e_node/dra_test.go

```diff
@@ -382,6 +384,43 @@ var _ = framework.SIGDescribe("node")("DRA", feature.DynamicResourceAllocation,
 			gomega.Eventually(kubeletPlugin2.GetGRPCCalls).WithTimeout(retryTestTimeout).Should(testdriver.NodeUnprepareResourcesSucceeded)
 		})
 
+		ginkgo.It("must provide metrics", func(ctx context.Context) {
+			kubeletPlugin1, kubeletPlugin2 := start(ctx)
+
+			pod := createTestObjects(ctx, f.ClientSet, getNodeName(ctx, f), f.Namespace.Name, "draclass", "external-claim", "pausepod", false, []string{kubeletPlugin1Name, kubeletPlugin2Name})
+
+			ginkgo.By("wait for pod to succeed")
```

Comment:
```suggestion
			ginkgo.By("Waiting for pod to succeed")
```
As recommended at https://onsi.github.io/ginkgo/#documenting-complex-specs-by

Reply:
You are right, but `dra_test.go` already deviates from that recommendation. To ensure consistency, I would have to update the complete file - I prefer not to do that and instead use the same style also in this PR.

@bart0sh: your preference?

Reply:
+1 for keeping the same style